public/index.html

```diff
@@ -1,20 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="utf8">
+  <meta charset="utf-8">
   <title>Lo-Dash Web Builder</title>
-  <meta name="viewport" content="width=device-width">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
   <link rel="stylesheet" href="css/normalize.css">
   <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-  <div class="outer">    
-    <h1 class="center"><a href="http://lodash.com/">Lo-Dash</a> <span>v2.4.1</span></h1>
+  <div class="outer">
+    <h1 class="center"><a href="https://lodash.com/">Lo-Dash</a> <span>v3.1.1</span></h1>
```

Comment:
Can the version be gotten from say lodash on the page's `_.VERSION`. Also include lodash on the page if it isn't :P.